opentelemetry-declarative-config/src/logs/processor_config.rs

```diff
@@ -0,0 +1,18 @@
+//! # Log Processor Configuration module
+//!
+//! This module defines the configuration structures for log processors
+
+use serde::Deserialize;
+use serde_yaml::Value;
+
+/// Configuration for Log Processors
+#[derive(Deserialize)]
+pub struct ProcessorConfig {
+    pub batch: Option<ProcessorBatchConfig>,
+}
+
+/// Configuration for Batch Log Processor
+#[derive(Deserialize)]
+pub struct ProcessorBatchConfig {
+    pub exporter: Value,
```

Comment:
This is more generic comment for all the config's - Storing the exporter/processor/reader as `serde_yaml::Value` isn’t ideal - it bypasses all type validation and defers errors to runtime. This should use a typed struct or enum so schema and type issues are caught during parsing, not during exporter creation.